test/tools/yulrun.cpp

```diff
@@ -65,7 +65,7 @@ std::pair<std::shared_ptr<Block>, std::shared_ptr<AsmAnalysisInfo>> parse(std::s
 	);
 	if (stack.parseAndAnalyze("--INPUT--", _source))
 	{
-		yulAssert(stack.errors().empty(), "Parsed successfully but had errors.");
+		yulAssert(!stack.hasErrors(), "Parsed successfully but had errors.");
```

Comment:
Would be best to check how it behaves on input that generates warnings. Try this:

```yul
{
    let x := 42
    switch x
        default {
            sstore(x, x)
        }
}
```

Also, to print those warnings.

Reply:
Before it failed and with this change it doesn't anymore. I will revert this.